### PR TITLE
Add "contact_selection" Content Type Resolver

### PR DESCRIPTION
**Pull request by TheCadien**
Add missing "contact_selection" Content Type Resolver
See: https://github.com/sulu/SuluHeadlessBundle/issues/3

### CODE REVIEW

File: Content/ContentTypeResolver/ContactSelectionResolver.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        $content = [];
+        foreach ($this->contactManager->getByIds($data, $locale) as $contact) {
+            $serializationContext = new SerializationContext();
+            $serializationContext->setGroups(['partialContact']);
+
+            $content[] = $this->contactSerializer->serialize($contact->getEntity(), $locale, $serializationContext);
+        }
+
+        return new ContentView($content, [$data]);
```
Comment (alexander-schranz) [inline Content/ContentTypeResolver/ContactSelectionResolver.php:61]: This should not be nested again in an array:

```suggestion
return new ContentView($content, $data ?: []);
```

See [CategoryResolver](https://github.com/sulu/SuluHeadlessBundle/blob/cf25058ba6ed7f45a6644694981a18b5bae1210b/Content/ContentTypeResolver/CategorySelectionResolver.php#L62) as example.

File: Tests/Unit/Content/ContentTypeResolver/ContactSelectionResolverTest.php [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+                        'id' => 2,
+                        'formatUri' => '/media/2/{format}/media-2.jpg?v=1-0',
+                    ],
+                ],
+            ],
+            $result->getContent()
+        );
+
+        $this->assertSame(
+            [
```
Comment (alexander-schranz) [inline Tests/Unit/Content/ContentTypeResolver/ContactSelectionResolverTest.php:110]: We should be consistent here to the other multi selection and not nest again into an array.

File: Content/ContentTypeResolver/ContactSelectionResolver.php (hunk 1 above)
Comment (TheCadien) [reply Content/ContentTypeResolver/ContactSelectionResolver.php]: Add it :)

File: Tests/Unit/Content/ContentTypeResolver/ContactSelectionResolverTest.php (hunk 2 above)
Comment (TheCadien) [reply Tests/Unit/Content/ContentTypeResolver/ContactSelectionResolverTest.php]: Add it :)

**Comment by alexander-schranz** [comment]
@TheCadien Thank you 👍

**Pull request merged**